lib/rabbitmq/cli/ctl/commands/decode_command.ex

```diff
@@ -43,13 +51,7 @@ defmodule RabbitMQ.CLI.Ctl.Commands.DecodeCommand do
     end
   end
 
-  def switches() do
-      [
-        cipher: :atom,
-        hash: :atom,
-        iterations: :integer
-      ]
-  end
+  use RabbitMQ.CLI.Core.RequiresRabbitAppRunning
```

Comment:
The decode command does not need the rabbit app to be running. It does not even need a node to be running.